runtime/vm/visible.c

```diff
@@ -118,7 +118,12 @@ checkVisibility(J9VMThread *currentThread, J9Class* sourceClass, J9Class* destCl
 #endif /* JAVA_SPEC_VERSION >= 11 */
 		} else if (modifiers & J9AccPrivate) {
 			/* Private */
-			if (sourceClass != destClass) {
+			BOOLEAN sameClass = (sourceClass == destClass);
```

Comment:
nitpick: may be more clear this way

`BOOLEAN sameClass = (sourceClass == destClass) || J9_ARE_J9CLASSES_COMPANIONS(sourceClass, destClass)`

Reply:
Fixed. Using the new macro J9_ARE_J9CLASSES_SAME().